float32 pick_X
float32 pick_Y
float32 pick_Z
float32 place_X
float32 place_Y
float32 place_Z
bool gripper_open
